config/install/easy_breadcrumb.settings.yml

```diff
@@ -1,26 +1,41 @@
 applies_admin_routes: false
 include_home_segment: false
+alternative_title_field: 'field_breadcrumb_title'
 home_segment_title: Home
 home_segment_keep: false
 include_title_segment: false
 language_path_prefix_as_segment: false
 use_menu_title_as_fallback: true
 use_page_title_as_menu_title_fallback: true
+menu_title_preferred_menu: ''
```

Comment:
This new setting is in the latest dev version but hasn't yet been included in a module release.
```suggestion
```